### ISSUE TITLE
missing options in documentation.

### ISSUE THREAD

**Issue by tony-gutierrez** [amqp/rhea#108]
The "transport" and "hostname" options are not in the docs at https://github.com/amqp/rhea#container under the connect method. "hostname" may be equiv to "servername"? My connection will not work without the "hostname" option. 

"idle_time_out" is also missing, and incorrectly documented in the code as:
```js
 /**
   * @property {number} [idle_time_out] The largest frame size that the sending
   * peer is able to accept on this connection.
   */
  idle_time_out?: number;
```

**Comment by amarzavery** [comment]
Fixed the documentation for idle_time_out. 

Not sure if hostname is same as servername. @grs - Thoughts?
PR #114 for other problems reported in this issue.

**Comment by grs** [comment]
servername is the SNI to use and may be different from hostname (though will default to host if not set explicitly).

**Comment by amarzavery** [comment]
fixed in #114
